products/contao.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Contao
+category: server-app
+permalink: /contao
+iconSlug: NA
+releasePolicyLink: https://contao.org/release-plan.html
+changelogTemplate: "https://github.com/contao/contao/blob/__RELEASE_CYCLE__/CHANGELOG.md"
+releaseColumn: true
+releaseDateColumn: true
+activeSupportColumn: true
+
+auto:
+  - git: https://github.com/contao/contao.git
+
+releases:
+  - releaseCycle: "5.0"
+    releaseDate: 2022-08-18
+    support: 2023-02-14
+    eol: 2023-02-14
+    latest: "5.0.7"
+    latestReleaseDate: 2022-11-30
+    lts: false
+    discontinued: false
```

Comment:
`discontinued` properties must be removed. It is reserved mainly for devices, and `discontinuedColumn` is not visible so it is never used.

Reply:
Done (@bezin sorry I had to override you commit to avoid conflicts).